stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/experimental/global/GlobalState.kt

```diff
@@ -0,0 +1,96 @@
+package io.getstream.chat.android.offline.experimental.global
+
+import io.getstream.chat.android.client.errors.ChatError
+import io.getstream.chat.android.client.models.ChannelMute
+import io.getstream.chat.android.client.models.Mute
+import io.getstream.chat.android.client.models.TypingEvent
+import io.getstream.chat.android.client.models.User
+import io.getstream.chat.android.offline.model.ConnectionState
+import io.getstream.chat.android.offline.utils.Event
+import kotlinx.coroutines.flow.StateFlow
+
+/**
+ * Global state of the Chat domain.
+ */
+public interface GlobalState {
+
+    /** The current user on the chatDomain object */
```

Comment:
Let's add missing @return parameters and puncations in all those comments

Reply:
These are variables and not functions. Should we add `@return` to them?